packages/core/src/controllers/AccountController.ts

```diff
@@ -1,4 +1,4 @@
-import { subscribeKey as subKey } from 'valtio/utils'
+import { subscribeKey as subKey } from 'valtio/vanilla/utils'
```

Comment:
Hmm, @glitch-txs curious, is this the line that removes some dependencies? If yes, that's awesome.

Reply:
The other one requires React because it's also imported internally, so it was throwing an error with yarn because it wasn't installing React. This path works the same way but the React dep is not imported internally.